WhisperAPI/WhisperAPI/log4net.config

```diff
@@ -0,0 +1,17 @@
+﻿<log4net>
+  <root>
+    <level value="ALL" />
+    <appender-ref ref="file" />
+  </root>
+  <appender name="file" type="log4net.Appender.RollingFileAppender">
+    <file value="logs/WhisperAPI.log" />
+    <appendToFile value="true" />
+    <rollingStyle value="Size" />
+    <maxSizeRollBackups value="5" />
+    <maximumFileSize value="10MB" />
+    <staticLogFileName value="true" />
+    <layout type="log4net.Layout.PatternLayout">
+      <conversionPattern value="%-5p%d{ yyyy-MM-dd HH:mm:ss} RequestId: %property{requestId} [%thread] %logger - %message%newline" />
```

Comment:
I prefer the message on a new line (on its own line). This is mainly because a lot of messages will take more than one line (when we have errors and stack traces for instance). I find it easier when every message starts on a new line (and the header with timestamp, request ID above).

Reply:
completely agree with that